charts/finrem-ccd-definitions/values.preview.template.yaml

```diff
@@ -331,11 +337,11 @@ ccd:
     imageTag: "ccd-divorce-logstash-latest"
     logstashJavaOpts: -Xmx1g -Xms512M
     extraEnvs:
-      - name: DB_URL
-        value: jdbc:postgresql://${SERVICE_NAME}-postgresql:5432/data-store?ssl=disable&stringtype=unspecified
-      - name: DB_USER
+      - name: DATA_STORE_PASS
```

Comment:
This need to be set as secret - example - 
https://github.com/hmcts/sscs-tribunals-case-api/blob/e3fdd3fdd32c3e0e99195b4f08f774af1dcf64b6/charts/sscs-tribunals-api/values.ccd.preview.template.yaml#L596-L599